cg/models/demultiplex/flow_cell.py

```diff
@@ -126,6 +126,21 @@ def sequencer_type(
         """Return the sequencer type."""
         return sequencer_types[self.machine_name]
 
+    def get_bcl_converter(self, bcl_converter: str) -> str:
+        """
+        Return the BCL converter to use.
+        Tries to get the BCL converter from the sequencer type if not provided.
+        """
+        return bcl_converter or self.get_bcl_converter_by_sequencer()
```

Comment:
Oh, is the bcl conversion software used really dependent on the sequencer type? Neat!

Reply:
I am not entirely sure, but if it is, we can remove the bclconvert parameter. @karlnyr may know

Reply:
So, the FlowCellDirectory data is used in both the Demultiplex and DemultiplexPostProcessing API. 

They are used in a different capacity in these two places where the BCL converter parameter has an impact. 

It can be so that we want to demultiplex a novaseq flow cell with bcl2fastq (for reproducability) and this would not be possible if we were to remove the bcl converter option here since it would always default to "dragen" or "bclConvert" for that matter.

Reply:
I think we should leave it then